docs/samples/Microsoft.ML.Samples.GPU/Program.cs

```diff
@@ -12,7 +12,7 @@ internal static void RunAll()
         {
             int samples = 0;
             var types = Assembly.GetExecutingAssembly().GetTypes();
-            foreach (var type in types)
```

Comment:
This file should be reverted to run all samples again

Reply:
Reverted.